lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyConstants.java

```diff
@@ -74,4 +76,6 @@ private LibertyConstants() {
 
     public static String changedFeatureNameDiagMessage="ERROR: The %s feature cannot be configured with the %s feature because they are two different versions of the same feature. " +
             "The feature name changed from %s to %s for Jakarta EE. Remove one of the features.";
+
+    public static List<String> filesToWatch= Arrays.asList(SERVER_XML,"server.env","bootstrap.properties");
```

Comment:
We actually care about all xml files since the server.xml can include other xml files. There are several folders that can contain xml files used for server configuration. See this [doc](https://openliberty.io/docs/latest/reference/config/server-configuration-overview.html#server-xml).

Reply:
We are watching other config files separately using LibertyUtils.isConfigXMLFile, which include pattern
".*/(?:usr/shared/config/.+|configDropins/(?:defaults|overrides)/.+|server\\.xml)$";